src/content/blog/whats-new-december-2023.mdx

```diff
@@ -0,0 +1,162 @@
+- Updating our integration docs to include MDX components and Expressive Code
+
+## Community
+
+### Community Awards November
+
+- 🌟 $250 awarded to Atharva for significant contributions to Astro Docs and active community involvement.
+- 🥳 $250 awarded to McFlyPartages for translating a significant portion of the Astro Docs into French.
+- 🎉 $250 awarded to Hippo for exceptional work releasing, maintaining, and providing support for Expressive Code.
+- 🍾 $250 awarded to Eva Decker for actively supporting the Astro Community on Discord.
+- 🎊 $250 awarded to Jumper for being a super positive, welcoming and supportive presence in the Astro Discord.
+- 🎁 $250 awarded to Happydev for significant work on the Astro Core code base.
+
+Check them out or learn more about the Community Awards on [our community website](https://community.astro.build).
+
+## Showcase
+
+### Content
+
+🚉  See how a full site migration went in [Nuxt 2 to Astro 3 Replatforming – from Setup to Production](https://stevenwoodson.com/blog/replatforming-from-nuxtjs-2-to-astro/) by @stevenwoodson
+
+🧑‍🍳 Follow the recipe for [Quick-and-useful: A DIY, self-hosted Linktree and Linkinbio clone with Astro and TinaCMS](https://darko.io/posts/linktree-and-linkinbio-clone-with-astro-and-tinacms/) by @\_seiryuu\_
+
+⚠️ Get some tips, because maybe [You are using Astro wrong](https://arshx.substack.com/p/you-are-using-astro-wrong-a-survival) by @arshxcx
+
+🚀 Welcoming a new astronaut to the spaceship! A great article to send to your Astro-curious friends: [Launching a blog with Astro](https://goulven-clech.dev/2023/launching-blog-astro) by @goulvenclech
+
+📧 Learn how to use [using RESEND (email API) with Astro](https://www.youtube.com/watch?v=HyDwVN1AFwY&t=2s) in a new video by @chrispennington
+
+🎉 Read all about [A year working with HTML Web Components](https://hawkticehurst.com/writing/a-year-working-with-html-web-components/) by @hawkticehurst
+
+👀 Astro's 6-hour Launch Week Community Call Deluxe is on [YouTube](https://www.youtube.com/watch?v=VqfzAkI4AkY) hosted by @sarah11918 
+
+🛠️ Want to move [from Jekyll to Astro](https://jackcarey.co.uk/posts/astro-rewrite/)? Read this post by @jack_jack_jack
+
+🤿 Take a [Deep dive on View Transitions blog post series](https://fryuni.dev/deep-dive/astro-view-transitions/) by @Fryuni
+
+🏎️ New to Astro? Check out this updated [Astro Crash Course](https://www.youtube.com/watch?v=XoIHKO6AkoM) by Brad Traversy
+
+🏁 Define the controlled rollout of your next feature by following [Adding feature flags in Astro with Xata (showcasing View Transitions Forms)](https://xata.io/blog/feature-flags-tutorial) by @rishirajjain
+
+### Themes
+
+New Astro themes released this month. Try them out!
+
+- [Docbox](https://docbox-astro.vercel.app/) by @tffarhad
+- [ltree theme for Astro](https://github.com/DBozhinovski/astro-theme-ltree)
+- [Brutal](https://brutal.elian.codes) has reached over 150 stars by @eliancodes
```

Comment:
MOAR THEMES!

```suggestion
- [Dusk](https://lexingtonthemes.com/info/dusk/) by @michael-andreuzza
- [Brutal](https://brutal.elian.codes) has reached over 150 stars by @eliancodes
```

Reply:
MOAR:

- [Indie Astro SSR starter](https://petitpois.gumroad.com/l/indie-astro/6n6943w) by @petitpois

Reply:
Nice! Updated my suggestion to include this too.

Reply:
Thanks!